include/exadg/convection_diffusion/preconditioners/multigrid_preconditioner.cpp

```diff
@@ -281,11 +281,14 @@ template<int dim, typename Number>
 void
 MultigridPreconditioner<dim, Number>::set_velocity(VectorTypeMG const & velocity)
 {
+  unsigned int const fine_level   = this->get_number_of_levels() - 1;
```

Comment:
I `get_number_of_levels` checking that the returned value is `>0`?

Reply:
I can add this.

Reply:
Has already been done. see implementation of function.